engine/combat/utility/sos_appraisal.py

```diff
@@ -103,9 +110,25 @@ def execute_selecting_command(self: Self) -> None:
         ):
             sos_ctrl().dpad.tap_down()
         else:
-            self.step = SoSAppraisalStep.ConfirmCommand
+            self.step = SoSAppraisalStep.Boost
             logger.debug(f"Selecting Battle Command: {self.battle_command.name}")
 
+    def execute_boost(self: Self) -> None:
+        # TODO(eein): Check if theres a flag that shows when live mana is available and use
+        # that to guard this function as well
+        if self.boost > 0:
+            logger.debug(f"Boosting: {self.battle_command.name} {self.boost} times")
+            sos_ctrl().toggle_boost(True)
+            time.sleep(0.5)
+
+            for boost in range(self.boost):
+                logger.debug(f"Triggering Boost: {boost + 1}")
+                sos_ctrl().confirm()
+                time.sleep(0.2)
```

Comment:
This one is not necessary, can use `sos_ctrl().confirm(tapping=True)` instead, which will inject a sleep after the release.

Doesn't fix the issue I mentioned above, but hides it away somewhat :P